src/main/resources/com/google/api/codegen/java/common.snip

```diff
@@ -158,7 +158,7 @@
 @end
 
 @snippet resourceName(collectionConfig)
-  {@context.lowerUnderscoreToUpperCamel(collectionConfig.getEntityName)}Name
+  {@context.getResourceName(collectionConfig.getEntityName)}
```

Comment:
Don't add a context method for this (it will create more trouble for my refactoring)